docs/production-deployment/cloud/account-setup/users.mdx

```diff
@@ -259,59 +272,59 @@ Account Global Admin has Namespace Admin permissions on Namespaces.
+| DeleteUser                            |                    |           |               | ✔            | ✔             |
+| GetAccount                            | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetAccountFeatureFlags                | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetAccountLimits                      | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetAccountSettings                    | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetAccountUsage                       |                    |           |               | ✔            | ✔             |
+| GetAPIKey                             | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetAPIKeys                            | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetAsyncOperation                     | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetDecodedCertificate                 | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetIdentities                         | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetIdentity                           | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetNamespaces                         | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetNamespacesUsage                    |                    |           |               | ✔            | ✔             |
+| GetRegion                             | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRegions                            | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRequestStatus                      | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRequestStatuses                    |                    |           |               | ✔            | ✔             |
+| GetRequestStatusesForNamespace        | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRequestStatusesForUser             | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRoles                              | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetRolesByPermissions                 | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetServiceAccount                     | ✔                  | ✔         | ✔             | ✔            | ✔             |
+| GetServiceAccounts                    | ✔                  | ✔         | ✔             | ✔            | ✔             |
```

Comment:
There are caveats with apis like this that we should eventually try to add documentation for.

For example I believe this api is part of the Read-Only Role but we have extra checks to ensure only namespace admins can manage namespace scoped service accounts